### PR TITLE
Fontconfig for Android Studio

### PR DESCRIPTION
**Pull request by phunehehe**
###### Motivation for this change

With the fonts.conf by Ubuntu (I use Nix instead of Apt on Ubuntu), Android Studio crashes during startup because it [fails to do some font stuff](https://gist.github.com/phunehehe/f3389bc79b4428a03b20053377ebdd5a). With a config file built with `makeFontsConf` (even an empty one!?), it works fine. The workaround is found via https://github.com/docker-library/openjdk/issues/73.

On the one side, it could be argued that I should fix my fonts.conf, but then it means the package isn't really usable out of NixOS.

On the other side, overriding FONTCONFIG_FILE probably means the package can't use all the installed fonts, but then I also see other packages doing the same.

Thoughts?

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -12246,19 +12246,10 @@ in
...
-  # download. OpenJDK is straightforward, but may suffer from compatibility
-  # problems e.g. https://code.google.com/p/android/issues/detail?id=174496.
-  # To use Oracle JDK add an override to ~/.nixpkgs/config.nix:
-  # {
-  #   packageOverrides = pkgs: {
-  #     android-studio = pkgs.android-studio.override {
-  #       jdk = pkgs.oraclejdk8;
-  #     };
-  #   };
-  # }
```
Comment (grahamc) [inline pkgs/top-level/all-packages.nix]: Why delete this comment?

Comment (phunehehe) [inline pkgs/top-level/all-packages.nix]: Because it's no longer true. This version of android studio bundles its own JDK.

**Comment by grahamc** [comment]
Is this still good to merge?

**Comment by phunehehe** [comment]
It is. The only concern I had was about fontconfig. If nobody chips in I guess it's not really a problem.

**Comment by grahamc** [comment]
If someone is impacted by this, change, hopefully they'll file an issue :)

**Pull request merged**